src/test/ui/const-generics/normalization_failure.rs

```diff
@@ -0,0 +1,16 @@
+// run-pass
```

Comment:
Can you name this test `issue-70125-2.rs`? It's just a reduced version of the original testcase.